### ISSUE TITLE
Prisoner's Dilemma explanation incomplete

### ISSUE THREAD

**Issue by hrvojesimic** [ncase/trust#48]
At the beginning of the story we are introduced to a classic Donation Game, a special case of (non-iterated) Prisoner's Dilemma (PD). The explanation is great for the most part. However, i feel there are two large holes in the explanation that will probably confuse people that are new to this game:

## 1. The goal of the game

The goal of the game is to *maximise your score*. That's it. The goal is not to be nice, or to have more points that the other player, or to maximise overall score of all players. The only goal for every single player is to maximise its own score (however, their strategies for doing so may be completely different, some may even be irrational).

For example, the story goes:
> Because if you both cooperate, you both give up a coin to gain three. (score: +2 vs +2) But if you cheat & they cooperate, you gain three coins at their cost of one. (score: +3 vs -1) Therefore: you "should" still CHEAT.

This is true, but there are subtle issues with this explanation:

- When considering your best move in PD, it makes no sense to discuss opponent's payout. It is irrelevant for us if they will get a +2 or -1 or +million. Since most people expect your Game of Trust to be a zero-sum game, where opponent's payout matters very much, this can be confusing. PD is not a zero-sum game.
- There should be no quotes around "should" if we want to explain the game. Three is bigger than two, so you should cheat. PD is not an ethics issue, it is a game that is a logical precursor of ethics - explaining it the other way around, saying what you *should* and "should" do, breeds confusion.

This would be more to the point:
> Because if you both cooperate, you will give up a coin to gain three. (score: +2) But if you cheat & they cooperate, you gain three coins without giving up anything. (score: +3) Three is bigger than two, therefore you should still CHEAT.

## 2. The problem with PD

The problem is not that cheaters win, the problem is that there is a *suboptimal Nash equilibrium* in the non-iterated version that yields zero score instead of +2. By doing what's in their best interest (cheating), players end up hurting their bottom line. If players could vote to abolish cheating they would do it, because that would lead to greater personal gain.

For example, you write:
> Trust is nice, but it can let others take advantage of you -- or shoot you as you come unarmed out of a trench. Sometimes, distrust is rational!

It's to early to talk about trust before the iterated version. Trust in other side solves nothing: if you *knew* that the other player will cooperate, you are *still* better off cheating.

I propose something like this:
> But, if everybody cheats, we will never receive any coins from this wonderful machine! We are incapable of forcing the other side to cooperate, so instead of everyone earning two coins per round, we're stuck at zero.

**Comment by aliqq** [comment]
I completely agree with the above (but think the explanations shouldn't get more complicated), and would like to add a similar issue in the "evolution" section. There it says

> Game theory has two powerful ideas about this: "Zero-sum game". This is the sadly common belief that a gain for "us" must come at a loss to "them", and vice versa. "Non-zero-sum game". This is when people make the hard effort to create a win-win solution! (or at least, avoid a lose-lose) Without the non-zero-sum game, trust cannot evolve. 

The phrasing suggests that the game is zero sum, or maybe that a whether a game is zero sum depends on the player's beliefs or actions. To me it also sounded as if it's non-zero sum, but if I do the change of +2/+2 to +1/+1 on the previous page it becomes zero-sum. All of this is of course not true, it remains being a PD (unless you go for something like +0/+0, +1/-1, +0+0, which is nowhere discussed).

I would propose to rephrase the page completely, to something like:
> As long as players believe to be in a zero-sum game, where every profit for other players comes out of their pocket, then mistrust and non-cooperation spreads. Even if the game really rewards everyone with win-win opportunities.

(Reason behind this comment: I shared the link with my team because I love the easy-to-understand implications for cooperative behavior, and instead of talking about win-win I ended in a discussion about zero-sum vs. PD)

**Comment by hrvojesimic** [comment]
I agree that introducing zero-sum game at this point does not help. However, the issue is not false belief of players. Rather, in PD there must be a *substantial* reward for cooperation to counterbalance the risky behaviour of being "nice". So I would say something like this:

> Even when the payoffs are positive on balance, the reward for cooperation can be too small to compensate the risk of trying to create a *win-win* outcome. Without a substantial reward for cooperation on both sides, *trust cannot evolve*.

EDIT: Removed the part about zero-sum. It's best to dispense altogether with the explanation of "zero-sum" vs "non-zero-sum" games. They complicate without helping us understand why some IPDs don't foster cooperation.
